src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -1085,7 +1087,8 @@ private void duplicateCurrentlySelectedModElement() {
 								return new ModElementNameValidator(mcreator.getWorkspace(), (VTextField) component,
 										L10N.t("common.mod_element_name")).validate();
 							}
-						}, L10N.t("workspace.elements.duplicate"), UIManager.getString("OptionPane.cancelButtonText"));
+						}, L10N.t("workspace.elements.duplicate"), UIManager.getString("OptionPane.cancelButtonText"),

```

Comment:
If possible, keep those unrelated changes out of the PR. I will run global code reformat in the near future

Reply:
WDYM, this change adds the breadcrumb to the dialog, doesn't it?

Reply:
![image](https://github.com/MCreator/MCreator/assets/16374228/9e8aea74-96a3-4457-ad3b-45f81ea91cc9)

This change splits one line into two

Reply:
It is isolated from the change that adds to the dialog

Reply:
But can it fit into one line?